src/ElvAccount.js

```diff
@@ -119,9 +119,7 @@ class ElvAccount {
       await client.userProfileClient.CreateWallet();
 
       if (tenantId) {
-
-        await this.SetAccountTenantContractId({tenantId});
-
+        await client.userProfileClient.SetTenantContractId({tenantContractId: tenantId});
```

Comment:
Should we remove `this.SetAccountTenantContractId` ?

Reply:
other methods are using it, dont want to disturb them for now.